### ISSUE TITLE
Unity Editor enters an unresponsive black screen with the simulator

### ISSUE THREAD

**Issue by nniranjhana** [lgsvl/simulator#1109]
I followed the instructions on https://www.lgsvlsimulator.com/docs/build-instructions/ and when I click on the project simulator, it loaded up a black screen on the Unity Editor and I'm unable to do step 12, as this window (attached image below) won't close (but it minimizes) and it is unresponsive if I click on any options (for Simulator > Build WebUI..).

I tried creating a new project (not adding the cloned simulator) and it worked fine. I rebooted and I still had the same problem. How do I fix this issue?

![unresponsive-unity-editor](https://user-images.githubusercontent.com/29974283/98087271-ac226500-1e34-11eb-8d65-f6fe95cd6f6d.png)

**Comment by EricBoiseLGSVL** [comment]
First off download and install Unity 2019.3.15f1 for the 06 release [here](https://unity3d.com/get-unity/download/archive).  Be sure to include Windows modules.
Close unity and delete the Library folder in the root of the cloned repo.  Open Unity and let it create the folder.  Close Unity after it is done and open it again.
If you still have issues, please find the Editor player log [here](https://docs.unity3d.com/Manual/LogFiles.html) and post.

**Comment by nniranjhana** [comment]
I followed your instructions, unfortunately the problem still persists. PFA the Editor log.

[Editor.log](https://github.com/lgsvl/simulator/files/5496093/Editor.log)

**Comment by EricBoiseLGSVL** [comment]
This is your issue
```
Vulkan API version 1.1.0 (1.2.131 supported by driver)
Vulkan vendor=[Intel] id=[8086]
Vulkan renderer=[Intel(R) UHD Graphics 630 (CML GT2)] id=[9bc5]
Vulkan device type 1
Vulkan driverversion=[0x05000008] uint=[83886088]
Vulkan PSO: cache data not found [path=/tmp/LG Silicon Valley Lab/LGSVL Simulator/vulkan_pso_cache.bin]
Initialize mono
```

You need to be sure Simulator is using the GPU.  Please see the linked issues here #1057

**Comment by nniranjhana** [comment]
Thanks for pointing me to the root of the issue! I checked #1057, #1076, #879, #753 which were all related. But I could not find the exact solution there (I had already added the PPA, my NVIDIA drivers installed, `libvulkan1` and `vulkan-tools` installed (but vkcube didn't render) and I did not want to run LGSVL in a docker container to solve this).

So on Ubuntu, I had to open NVIDIA X Server Settings, go to PRIME Profiles tab, and choose NVIDIA On-Demand (it was previously in Performance Mode) and then reboot. I am now able to work with the simulator on a responsive Unity Editor.

For anyone else who might need it in the future:

https://askubuntu.com/questions/1201072/how-nvidia-on-demand-option-works-in-nvidia-x-server-settings

![nvidia-change-settings](https://user-images.githubusercontent.com/29974283/98299672-b13efb80-1f7d-11eb-81c4-069ab87a0f0f.png)
